html/janus.js

```diff
@@ -1283,6 +1289,9 @@ function Janus(gatewayCallbacks) {
 						    'optional': []
 						};
 					}
+					if(typeof media.video === 'object') {
+						videoSupport = media.video;
```

Comment:
But then shouldn't this be BEFORE checking lowres, hires, etc.? This way, we're still autodetecting, and then overwriting.